R/convertToMAE.R

```diff
@@ -0,0 +1,72 @@
+#' Wrapper that converts an input list into a MultiAssayExperiment object
+#' 
+#' @details This function takes in a list of key-value pairs (keys: data types,
+#' values: matrices/dataframes) and calls the necessary functions from the
+#' MultiAssayExperiment and SummarizedExperiment packages to incorporate the
+#' values from the input list into a MultiAssayExperiment object, transforming
+#' the values according to the keys. 
+#' @param dataList (list) input key-value pairs (keys: data types, values: data in
+#' the form of matrices/dataframes); must have a key-value pair that
+#' corresponds to patient IDs/metadata labelled patientPheno.
+#' @return MAE (MultiAssayExperiment) data from input list incorporated into a
+#' MultiAssayExperiment object, compatible with further analysis using the 
+#' netDx algorithm.
+
```

Comment:
Add a line with 
#' @export so the function is useable outside the package

Reply:
Added #' @export so that the function is useable outside the package.